spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/internal/spring/ArmeriaConfigurationUtil.java

```diff
@@ -188,6 +189,10 @@ public static void configureServerWithArmeriaSettings(
                                                       compression.getExcludedUserAgents(),
                                                       minBytesToForceChunkedAndEncoding));
         }
+
+        if (settings.enableAutoInjection()) {
+            server.dependencyInjector(SpringDependencyInjector.of(beanFactory), false);
```

Comment:
Didn't we decide to use the `DependencyInjector` registered as a `@Bean` instead of `SpringDependencyInjector` even though `enableAutoInjection` is set to `true`?

Reply:
Yeah, we need both. 😄